src/hotspot/share/jvmci/vmStructs_jvmci.cpp

```diff
@@ -479,10 +480,6 @@
   declare_constant(JVMCINMethodData::SPECULATION_LENGTH_BITS)             \
                                                                           \
   declare_constant(JVM_ACC_WRITTEN_FLAGS)                                 \
-  declare_constant(JVM_ACC_HAS_FINALIZER)                                 \
-  declare_constant(JVM_ACC_IS_CLONEABLE_FAST)                             \
-  declare_constant(JVM_ACC_IS_HIDDEN_CLASS)                               \
```

Comment:
`JVM_ACC_IS_HIDDEN_CLASS` and `JVM_ACC_IS_VALUE_BASED_CLASS` are actually used in the compiler (see [here](https://github.com/search?q=repo%3Aoracle%2Fgraal+path%3Acompiler+JVM_ACC_IS_HIDDEN_CLASS&type=code) and [there](https://github.com/search?q=repo%3Aoracle%2Fgraal+path%3Acompiler+JVM_ACC_IS_VALUE_BASED_CLASS&type=code)) so i think `KlassFlags::_misc_is_hidden_class` and `KlassFlags::_misc_is_value_based_class` also need to be exposed below along the other 2 bits.
/cc @mur47x111

Reply:
Yes, please add these two symbols as well
```
diff --git a/src/hotspot/share/jvmci/vmStructs_jvmci.cpp b/src/hotspot/share/jvmci/vmStructs_jvmci.cpp
index f0af57f9513..9d65268f0fe 100644
--- a/src/hotspot/share/jvmci/vmStructs_jvmci.cpp
+++ b/src/hotspot/share/jvmci/vmStructs_jvmci.cpp
@@ -727,6 +727,8 @@
                                                                           \
   declare_constant(InstanceKlassFlags::_misc_has_nonstatic_concrete_methods)   \
   declare_constant(InstanceKlassFlags::_misc_declares_nonstatic_concrete_methods) \
+  declare_constant(KlassFlags::_misc_is_hidden_class)                     \
+  declare_constant(KlassFlags::_misc_is_value_based_class)                \
   declare_constant(KlassFlags::_misc_has_finalizer)                       \
   declare_constant(KlassFlags::_misc_is_cloneable_fast)                   \
                                                                           \
```

Reply:
Ok, I added these flags in like this.  I didn't see them in the jdk code.  We'll have to coordinate the graal code and this change.  Also these flags are in misc_flags_offset() now.